db/sample_data/events.rb

```diff
@@ -74,7 +74,7 @@ def event_gongakrobatik
       end_date: Date.new(2021, 03, 05)
   )
   Event.new(
-      name: 'Einführung in die Kunst der Gongakrobatik',
+      name: 'Gongakrobatik',
```

Comment:
Why did you do this? Only for the presentation because long titles don't work well at the moment? If this is the case, please open a bug/technical debt/enhancement issue so we don't forget this (Since titles of that length shouldn't be a problem in the future 🙂 )